java/code/src/com/suse/manager/api/HttpApiRegistry.java

```diff
@@ -136,7 +139,7 @@ public static Set<String> getUnautenticatedRoutes() {
      */
     private void registerAuthEndpoints() {
         registrationHelper.addPostRoute(HTTP_API_ROOT + "auth/login", LoginController::apiLogin);
-        registrationHelper.addPostRoute(HTTP_API_ROOT + "auth/logout", withUser(LoginController::logout));
+        registrationHelper.addGetRoute(HTTP_API_ROOT + "auth/logout", withUser(LoginController::logout));
```

Comment:
Why was this changed from get to post? Will this influence the way other pages/locations call this method? Or is this endpoint only used by costumers? If that is the case, we would need to add something to release notes.

Reply:
I think it was a mistake in the initial HTTP API implementation. In XML-RPC it's `@ReadOnly` which means it's supposed to be a GET request in HTTP API (also no payload). That's why I changed it now. The change only affects the HTTP API (and the testsuite, which is included in the PR).

And yes, since it's on the public API, we'll need a release note.